source/tests/universal/dpmodel/atomc_model/test_atomic_model.py

```diff
@@ -398,3 +402,67 @@ def setUpClass(cls):
         cls.expected_has_message_passing = ds.has_message_passing()
         cls.expected_dim_fparam = ft.get_dim_fparam()
         cls.expected_dim_aparam = ft.get_dim_aparam()
+
+
+@parameterized(
+    des_parameterized=(
+        (
+            *[(param_func, DescrptDPA1) for param_func in DescriptorParamDPA1List],
+            *[(param_func, DescrptDPA2) for param_func in DescriptorParamDPA2List],
+            (DescriptorParamHybridMixed, DescrptHybrid),
+            (DescriptorParamHybridMixedTTebd, DescrptHybrid),
```

Comment:
Why there's test for se_a models in pt but not here in dp?

Reply:
I add it